devops/actions/build_container/action.yml

```diff
@@ -21,6 +21,9 @@ inputs:
   file:
     description: "Dockerfile"
     required: true
+  sycl_passwd:
```

Comment:
im not totally sure but this might be better off in the `secrets` section not the `inputs` section, see [here](https://docs.github.com/en/actions/sharing-automations/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow)

i might be wrong tho, imo if the `secrets` section works we should use that everywhere and only use `inputs`/`env` if we have to/are calling a third party action that requires that

Reply:
> im not totally sure but this might be better off in the `secrets` section not the `inputs` section, see [here](https://docs.github.com/en/actions/sharing-automations/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow)
> 
> i might be wrong tho, imo if the `secrets` section works we should use that everywhere and only use `inputs`/`env` if we have to/are calling a third party action that requires that

The thing is, that's not a re-usable workflow, but a standalone action, so I had to go this way in 4d28c776a08fbd7d66cb50eefc4acce579e5a597. Note that it is essentially the same as with `password` argument which also comes from a secret.

Reply:
ah ok, didnt know it would be different for actions vs workflows